static/app/utils/profiling/flamegraph.ts

```diff
@@ -148,7 +148,7 @@ export class Flamegraph {
         return;
       }
 
-      frames.unshift(stackTop);
```

Comment:
There was no need for unshift here - we were doing On for each frame we iterated over. I'm kinda hopeful this would have a pretty big impact on how long it takes to construct large profiles because the main thread sometimes blocks for quite a while 😢